extending-the-rest-api/modifying-responses.md

```diff
@@ -128,3 +128,88 @@ register_meta( $object_type, 'my_meta_key', $args1 );
 This example shows how to allow reading and writing of a post meta field. This will allow that field to be updated via a POST request to `wp-json/wp/v2/posts/<post-id>` or created along with a post via a POST request to `wp-json/wp/v2/posts/`.
 
 Note that for meta fields registered on custom post types, the post type must have `custom-fields` support. Otherwise the meta fields will not appear in the REST API.
+
+
+## Adding Links to the API Response
+WordPress generates a list of links associated with the queried resource to make it easier to navigate to related resources. The `embeddable` attribute controls whether the linked resource appears in the `_embedded` section of the repsonse.
```

Comment:
I'd split the `embeddable` bit to later on and describe links by themselves first. I'd also probably note something like "links are sent to the client as the `_links` property of the JSON object, but are stored separately internally from the response data"